webpack.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = (env, argv) => {
                     loader: 'babel-loader',
                     query: {
                         presets: [
-                            ['@babel/preset-env', {modules: false, targets: {safari: '7', ie: '10'}}],
+                            ['@babel/preset-env', {modules: false, targets: {chrome: '58', ie: '11', firefox: '58'}}],
```

Comment:
safari not a target anymore ?

Reply:
I've asked to romain and we support only those browsers :)

To go further, I think we can improve the babel configuration in order to use more of modern browser capabilities. Next guild-day perhaps ?